Release0.md

```diff
@@ -61,7 +61,7 @@ a Release as production-ready software in one of the next releases. We certainly
 hope to do so.
 
 To test how our k8s aaS platform will look like, have a look at
-https://github.com/SovereignCloudStack/k8s-cluster-api-provider
+[https://github.com/SovereignCloudStack/k8s-cluster-api-provider](https://github.com/SovereignCloudStack/k8s-cluster-api-provider)
```

Comment:
this link was already correct

Reply:
One can argue that our lint rules are in conflict with the Docusaurus v3 recommendations, and I don't really like the duplication (which is prone to errors and just doesn't look nice), but I don't want to press this matter too much. We can do as you said.

Reply:
You are right. That's why i would opt for meaningful link texts in square brackets as I just added.

Reply:
Nice! Thank you!